extension/chrome/elements/compose-modules/compose-recipients-module.ts

```diff
@@ -144,10 +144,15 @@ export class ComposeRecipientsModule extends ViewModule<ComposeView> {
   public validateEmails = (uncheckedEmails: string[]): { valid: EmailParts[]; invalid: string[] } => {
     const valid: EmailParts[] = [];
     const invalid: string[] = [];
+    const uniqueEmails = new Set<string>();
```

Comment:
Thanks, Roma - Upon further investigation, It wouldn't be possible to have duplicate emails, since Gmail itself prevents that.

I dig deeper into the code to answer my question why I got duplicate emails is that a draft were being made right before the recipients where added programmatically which produces such issue - so there is also that.

Later on, I have fixed the issue by moving the `startDraftTimer()` to which I believe where it should belong and that is on the `onRecipientAdded()` which requires web UI interaction, that way, I wouldn't get a blank draft with duplicate recipients when in fact opening a secure compose for the first time.